utilities/DateUtilities.ts

```diff
@@ -133,6 +133,13 @@ export class DateUtilities {
   ) {
     return arr.sort((a, b) => DateUtilities.gSorter(a, b, prop, desc));
   }
+
+  public static isWithin13Weeks(dateToCheck: Date): boolean {
+    const now = new Date();
+    const weeksAhead = new Date(now.getFullYear(), now.getMonth(), now.getDate() + 200);
```

Comment:
Is this right?
If it works the behaviour isn't intuitive to me, consider documenting or refactoring.